components/wb-bubble/bubble.js

```diff
@@ -0,0 +1,167 @@
+			$footer.addClass( componentName + "-mrgn" );
+
+			// Keep the bubble sticky while scrolling Y until user reaches the footer
+			var stickyUntilFooter = function( $element ) {
+
+				// Equals to bubble default bottom value in CSS
+				var bottomY = 30;
+				var $window = $( window );
+
+				if ( $window.scrollTop() >= $document.outerHeight() - $window.outerHeight() - $footer.outerHeight() ) {
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $document.outerHeight() - $window.outerHeight() - $window.scrollTop() ) + bottomY )
+					} );
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+			$( "main" ).after( "<footer id=\"wb-info\" class=\"" + componentName + "-mrgn" + "\"></footer>" );
+		}
+	},
+
+
+	setNotificationStatusWithExpiry = function( key, ttl ) {
+		var now = new Date();
+		localStorage.setItem( key, now.getTime() + ttl );
+	},
+
+	getNotificationStatusWithExpiry = function( key ) {
+		var expiryTimeValue = localStorage.getItem( key );
+		var now = new Date();
+
+		// compare the expiry time of the item with the current time
+		if ( now.getTime() > expiryTimeValue ) {
+
+			// If the item is expired, delete the item from storage
+			// and return null
+			localStorage.removeItem( key );
+			return null;
+		}
+		return expiryTimeValue;
+	},
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm,
+			avatar,
+			$bubbleElm,
+			$notification;
+
+		if ( elm ) {
+			$elm = $( elm );
+
+			// Set bubble avatar
+			avatar = $elm.data( componentName + "-avatar" );
+
+			if ( avatar ) {
+				$elm.css( "background-image", "url(" + avatar + ")" );
+			}
+
+			// Initiate dictionary
+			i18nDict = i18nDict[ wb.lang ] || i18nDict.en;
+
+			$bubbleElm = $( "<div class='" + componentName + "-wrap'></div>" );
+			$elm.wrap( $bubbleElm );
+
+			// Get notification status to check if it is still valid after TTL
+			isNotif = getNotificationStatusWithExpiry( componentName + "-notif" );
+
+			$notification = ( !isNotif ? "<p class='trans-left'>" +
+			"<span class='notif'>" + $elm.text() + "</span>" +
+			"<button class='notif-close' title='" + i18nDict.closeNotif + "' aria-label='" + i18nDict.closeNotif + "'>&times;</button></p>" : "" );
+
+			$elm.parent().append( $notification );
+			$elm.addClass( "trans-pulse" );
+
+			// Add link to the top page skip link.
+			wb.addSkipLink( i18nDict.skip + " " + $elm.text(), { "data-wb-doaction": $elm.attr( "data-wb-doaction" ) }, true );
```

Comment:
@Ricokola Can you add the following style in WET-BOEW for buttons in skip links?

```css
button.wb-sl:focus {
    width: auto;
    border: none;
    margin: 0;
}
```
This will prevent the skip link button from taking the full width and being bigger than it should.